gfsphysics/GFS_layer/GFS_typedefs.meta

```diff
@@ -4602,23 +4602,23 @@
   type = real
   kind = kind_phys
 [in_nm]
-  standard_name = in_number_concentration
-  long_name = IN number concentration
-  units = kg-1?
+  standard_name = ice_nucleation_number
```

Comment:
Same as for ccpp-physics, has this change been coordinated with @grantfirl and @ligiabernardet?